public/robots.txt

```diff
@@ -4,4 +4,21 @@
 # User-agent: *
 # Disallow: /
 
-Sitemap: https://repository.lib.duke.edu/sitemaps/repository.xml
+# Prevent all bots from crawling facet links
+User-agent: *
+Disallow: /catalog/facet/*
+Disallow: *f%5B*_facet_*%5D%5B%5D*
```

Comment:
@seanaery this won't match all facet fields, e.g. collection_title_ssi, admin_set_title_ssi. There may be other patterns I'm missing.

Reply:
@corylown Yes, good point. I think `collection_title_ssi` would be a safe / wise addition to the filter. Some facet parameters might not be, i.e., the savings on crawl traffic might not be worth the increased risk that important resources might go uncrawled by Google. E.g,.:
- admin_set_title_ssi is present in the admin set landing pages (https://repository.duke.edu/catalog?f%5Badmin_set_title_ssi%5D%5B%5D=Duke+Research+Data&q=&search_field=all_fields)
- active_fedora_model_ssi is present in any Browse Items or Browse collections link (https://repository.duke.edu/dc?f%5Bactive_fedora_model_ssi%5D%5B%5D=Item)

OK if I just add `collection_title_ssi` for now and then see how this plays out in production before revising further?